design/account.go

```diff
@@ -109,6 +109,7 @@ var _ = a.Resource("user", func() {
 		a.Response(d.BadRequest, JSONAPIErrors)
 		a.Response(d.InternalServerError, JSONAPIErrors)
 		a.Response(d.Unauthorized, JSONAPIErrors)
+		a.Metadata("swagger:extension:x-tag", "current_user")
```

Comment:
Can you please change the file permissions back to `644`?

And why do we need this?

Reply:
@aslakknutsen has asked to name this endpoint differently.

Reply:
@kwk I meant why do we need `Metadata`?

Reply:
@jarifibrahim Still learning Goa, is there a better way to do this? I was following the examples i could find for this - metadata seemed to make sense:
1. https://goa.design/reference/goa/design/apidsl/
2. https://github.com/goadesign/goa.design/blob/master/content/blog/002-endpoints.md